client/my-sites/patterns/pages/category/index.tsx

```diff
@@ -0,0 +1,98 @@
+import { useLocale } from '@automattic/i18n-utils';
+import {
+	__experimentalToggleGroupControl as ToggleGroupControl,
+	__experimentalToggleGroupControlOption as ToggleGroupControlOption,
+} from '@wordpress/components';
+import { CategoryNavigation } from 'calypso/components/category-navigation';
+import DocumentHead from 'calypso/components/data/document-head';
+import { PatternsGetStarted } from 'calypso/my-sites/patterns/components/get-started';
+import { PatternsHeader } from 'calypso/my-sites/patterns/components/header';
+import { RENDERER_SITE_ID } from 'calypso/my-sites/patterns/controller';
+import { usePatternCategories } from 'calypso/my-sites/patterns/hooks/use-pattern-categories';
+import { usePatterns } from 'calypso/my-sites/patterns/hooks/use-patterns';
+import ImgGrid from './images/grid.svg';
+import ImgStar from './images/star.svg';
+import type { PatternGalleryFC } from 'calypso/my-sites/patterns/types';
+
+import './style.scss';
+
+type PatternsCategoryPageProps = {
+	category: string;
+	isGridView?: boolean;
+	patternGallery: PatternGalleryFC;
+};
+
+export const PatternsCategoryPage = ( {
+	category,
+	isGridView,
+	patternGallery: PatternGallery,
+}: PatternsCategoryPageProps ) => {
+	const locale = useLocale();
+
+	const { data: categories } = usePatternCategories( locale, RENDERER_SITE_ID );
+	const { data: patterns } = usePatterns( locale, category );
+
+	const categoryNavList = categories?.map( ( category ) => ( {
+		name: category.name || '',
+		label: category.label,
+		link: `/patterns/${ category.name }`,
+	} ) );
+
+	return (
+		<>
+			<DocumentHead title="WordPress Patterns- Category" />
+
+			<PatternsHeader
+				title={ category + ' patterns' }
+				description="Introduce yourself or your brand to visitors."
+			/>
+
+			{ categoryNavList && (
+				<CategoryNavigation
+					selectedCategory={ category }
+					buttons={ [
+						{
+							icon: ImgStar,
+							label: 'Discover',
+							link: '/111',
```

Comment:
```suggestion
							link: '/patterns',
```
Let's make this a link back to the `/patterns` home page. 

Again, if we don't add `LocalizedLink` in `CategoryNavigation`, then we need to remember to localize the link here.